tests/libres_tests/res/enkf/test_es_update.py

```diff
@@ -205,9 +206,13 @@ def test_that_posterior_has_lower_variance_than_prior_with_init_files(copy_case)
     df_default = GenKwCollector.loadAllGenKwData(ert, "default")
     df_target = GenKwCollector.loadAllGenKwData(ert, "target")
 
-    assert df_default["COEFFS:COEFF_A"].var() > df_target["COEFFS:COEFF_A"].var()
-    assert df_default["COEFFS:COEFF_B"].var() > df_target["COEFFS:COEFF_B"].var()
-    assert df_default["COEFFS:COEFF_C"].var() > df_target["COEFFS:COEFF_C"].var()
+    # We expect that ERT's update step lowers the "size" of the covariance
+    # of the parameters, where "size" can be measured by the determinant.
+    assert (
+        0
+        < np.linalg.det(df_target.cov().to_numpy())
+        < np.linalg.det(df_default.cov().to_numpy())
+    )
```

Comment:
👍 
This definitely holds for the population covariances that the ensemble/sample is drawn from. Because `0<det(I-KH)<1`. So it also definitely holds when ensemble size tends to infinity.
It should also definitely hold for finite ensemble sizes when `df_target` and `df_default` are the same sample, i.e. when `df_target` is only an update of the sample `df_default`, and not an entirely new sample from the population posterior.

Reply:
Thanks!